### ISSUE TITLE
:eraSubsidiaryLocation proposal

### ISSUE THREAD

**Issue by jeropadillaRNE** [Interoperable-data/ERA-Ontology-3.1.0#15]
In the matter of era:SubsidiaryLocation, a Subsidiary Location must be linked to a Primary Location and specifies in more detailed way one of the following:

- Physical part of a primary location
- Attribute of primary location
- Different coding of a primary location

When it is related to a physical part, this could be a track, siding, switch, platform, level crossing, bridge, tunnel, signal, etc. All the mentioned physical parts are represented in the ontology diagram as era:BasicObject. In my opinion the diagram is missing an optional connection (maybe as a parameter) between era:SubsidiaryLocation and the fundamental components represented by era:BasicObject. I also have doubts about the need to make a Subsidary Location an era:BasicObject

This change will allow to connect the Subsidiary Location to the technical characteristics of the basic objects when the Subsidiary Location represents a physical part of the Primary Location

![image](https://github.com/Interoperable-data/ERA-Ontology-3.1.0/assets/167861371/ee9bdcf2-2e27-40b1-9771-9ed49e56a6a6)

**Comment by gatemezing** [comment]
> the matter of era:SubsidiaryLocation, a Subsidiary Location must be linked to a Primary Location and specifies in more detailed way one of the following:
> 
> * Physical part of a primary location
> * Attribute of primary location
> * Different coding of a primary location
> 
> When it is related to a physical part, this could be a track, siding, switch, platform, level crossing, bridge, tunnel, signal, etc. All the mentioned physical parts are represented in the ontology diagram as era:BasicObject. In my opinion the diagram is missing an optional connection (maybe as a parameter) between era:SubsidiaryLocation and the fundamental components represented by era:BasicObject. I also have doubts about the need to make a Subsidary Location an era:BasicObject

Yes, we need to solve this aspect since some of the previous types of Subsidiary Locations are now Classes. 
I imagine to still have a class `era:SubsidiaryLocation ` and reflect on the other types of Subsidiary locations that are not (yet) in the diagram. For example, Metastations, Loading Points, Balise, etc.

**Comment by gatemezing** [comment]
Dear @jeropadillaRNE , see this update here https://linkedvocabs.org/data/era-ontology/3.1.0/doc/index-en.html#SubsidiaryLocation

**Comment by jeropadillaRNE** [comment]
Dear @gatemezing thank you for considering our proposal. We can see the change in the diagram and the inclusion of the description of the SubsidiaryLocation, which is very similar to the [JSG Sector Handbook](http://taf-jsg.info/?page_id=280).
But we are not sure still about the connection between SubsidiaryLocation and era:InfrastructureObject. We think it will be more correct to have it with era:BasicObject. An SL will not be related to an era:AggregateObject.

The main problem, I think, is that we lack a discussion between the JSG and ERA so that it is understood what use an SL has, what types exist, who assigns them, etc. It was not possible to discuss it in the PLC TWG, but I think it is important to have it before the SubsidiaryLocation class is described in the Application Guide.
